docs/azure/sdk/protocol-convenience-methods.md

```diff
@@ -0,0 +1,209 @@
+
+- [**Azure.Core**](/dotnet/api/overview/azure/core-readme) provides shared primitives, abstractions, and helpers for building modern Azure SDK for .NET client libraries. These libraries follow the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html) and use package and namespace names prefixed with 'Azure', such as [`Azure.Storage.Blobs`](/dotnet/api/overview/azure/storage.blobs-readme).
+
+- [**System.ClientModel**](/dotnet/api/overview/azure/system.clientmodel-readme) is a lower-level library that provides shared primitives, abstractions, and helpers for .NET service client libraries. The `System.ClientModel` library is a general purpose toolset designed to help build libraries for a variety of platforms and services, whereas the `Azure.Core` library is specifically designed for building Azure client libraries.
+
+> [!NOTE]
+> The `Azure.Core` library itself also depends on the `System.ClientModel` for various service building blocks. In the context of this article, the key differentiator for method patterns is whether a client library depends on `Azure.Core` or `System.ClientModel` directly, rather than through a nested dependency.
+
+The following table compares some of the request and response types used by protocol and convenience methods based on whether the library depend on `Azure.Core` or `System.ClientModel`:
+
+|Request or response concern  |Azure.Core  | System.ClientModel |
+|---------|---------|---------|
+|Request body     | `RequestContent`         | `BinaryContent`         |
+|Advanced options     | `RequestContext`         | `RequestOptions`         |
+|Raw HTTP Response     | `Response`         | `PipelineResponse`         |
+|Input model     | `IPersistableModel<T>`         | `IPersistableModel<T>`         |
+|Output model     | `IPersistableModel<T>`         | `IPersistableModel<T>`         |
+|Return type with output model     | `Response<T>`         | `ClientResult<T>`         |
+
+The sections ahead provide implementation examples of these concepts.
+
+## Protocol and convenience method examples
+
+The coding patterns and types used by client library protocol and convenience methods vary slightly based on whether the library depends on `Azure.Core` or `System.ClientModel`. The differences primarily influence the C# types used for handling request and response data.
+
+### Libraries that depend on Azure.Core
+
+Many Azure SDK client libraries depend on the `Azure.Core` library. For example, the [`Azure.AI.ContentSafety`](/dotnet/api/overview/azure/ai.contentsafety-readme) library depends on the `Azure.Core` library and provides a `ContentSafetyClient` class that exposes both protocol and convenience methods.
+
+#### Work with protocol methods
```

Comment:
What do you think about eliminating the "Work with protocol methods" and "Work with convenience methods" headings and replacing with "Protocol" and "Convenience" tabs? It would reduce the amount of vertical scrolling needed and therefore make this doc seem less intimidating.